packages/suite/src/components/suite/SuiteLayout/index.tsx

```diff
@@ -141,6 +142,27 @@ const BodyWide = ({ url, menu, appMenu, children, guideOpen }: BodyProps) => (
     </Body>
 );
 
+/**
+ * Displays menu and body in two columns when guide is closed.
+ * Displays menu inlined in body when guide is open.
+ */
+const BodyNormal = ({ url, menu, appMenu, children, guideOpen }: BodyProps) => (
+    <Body>
+        <Columns>
+            {!guideOpen && menu && <MenuSecondary>{menu}</MenuSecondary>}
+            <ScrollAppWrapper url={url}>
+                {guideOpen && menu}
+                {appMenu}
+                <DefaultPaddings>
+                    <MaxWidthWrapper>{children}</MaxWidthWrapper>
+                </DefaultPaddings>
+            </ScrollAppWrapper>
+            {guideOpen && <StyledGuidePanel />}
+        </Columns>
+    </Body>
+);
+
+/** Displays menu inlined in body and hides guide. */
 const BodyNarrow = ({ url, menu, appMenu, children }: BodyProps) => (
```

Comment:
```suggestion
const BodyMobile = ({ url, menu, appMenu, children }: BodyProps) => (
```

Reply:
Lets make it self-descriptive

Reply:
b64f41d4dea3b528d0a0f9c55122b624b9e14c14

I'll resolve this with your permission. .)